src/main/java/Helper/Helper.java

```diff
@@ -0,0 +1,47 @@
+package Helper;
+
+import DBAccesser.User.QueryUser;
+import DBAccesser.Chat.QueryChat;
+import DBAccesser.Message.QueryMessage;
+
+import java.util.Random;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Helper {
```

Comment:
Helper is a bad name choice for this class.

Let's call this something like `UniqueId` or `UniqueIdGenerator`.

Reply:
Done. What about the `Helper` package? It contains `UniqueIDGenerator` and `SuccessResponseGenerator`.